manifests/overlays/prod/applications/data_hub/dh-prod-trino.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: dh-prod-trino
+spec:
+  destination:
+    namespace: dh-prod-trino
+    server: https://api.ocp4.prod.psi.redhat.com:6443
+  project: data-hub
+  source:
+    path: trino/overlays/prod
+    repoURL: https://github.com/AICoE/idh-manifests.git
```

Comment:
https://github.com/AICoE/idh-manifests/tree/production/trino does not have `trino/overlays/prod` as an available path

Reply:
https://github.com/AICoE/idh-manifests/pull/132 is the PR to add trino prod overlay